src/components/VueSearchbox.jsx

```diff
@@ -86,14 +104,209 @@ const VueSearchbox = {
           fuzziness,
           searchOperators
         });
+        this.searchBase.subscribeToStateChanges(this.setStateValue, [
+          "suggestions"
+        ]);
+
+        this.searchBase.onValueChange = nextValue => {
+          this.currentValue = nextValue;
+          this.$emit("valueChange", nextValue);
+        };
       } catch (e) {
         console.error(e);
       }
+    },
+    setStateValue({ suggestions = {} }) {
+      this.suggestionsList = (suggestions.next && suggestions.next.data) || [];
+    },
+    onInputChange(event) {
+      console.log("inside oninputchange", event);
+      this.setValue({ value: event.target.value, event });
+    },
+    onSuggestionSelected(suggestion) {
+      this.setValue({ value: suggestion && suggestion.value, isOpen: false });
+      this.searchBase &&
+        this.searchBase.triggerClickAnalytics(
+          suggestion && suggestion._click_id
+        );
+    },
+    setValue({ value, isOpen = true }) {
+      console.log("inside setValue", this.searchBase);
+      const { debounce } = this.$props;
+      this.isOpen = isOpen;
+      if (debounce > 0)
+        this.searchBase.setValue(value, { triggerQuery: false });
+      this.triggerSuggestionsQuery(value);
+    },
+    triggerSuggestionsQuery(value) {
+      console.log("inside triggerSuggestionsQuery", value);
```

Comment:
Let's remove all console statements. You can find using search and remove them all.